integration-tests/pom.xml

```diff
@@ -42,5 +42,9 @@
             <artifactId>hamcrest-library</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
```

Comment:
IS this addition actually required? We _shouldn't_ be using mocks in integration tests unless its unavoidable.